File: src/test/java/cn/enaium/cf4m/test/config/ConfigTest.java
package cn.enaium.cf4m.test.config;

import cn.enaium.cf4m.CF4M;
import cn.enaium.cf4m.provider.ConfigProvider;
import org.junit.jupiter.api.Assertions;

/**
 * @author Enaium
 */
public class ConfigTest {
    public ConfigTest() {
        Assertions.assertNotEquals(0, CF4M.CONFIG.getAll().size());
        for (ConfigProvider configProvider : CF4M.CONFIG.getAll()) {
            System.out.println(configProvider.getName() + "|" + configProvider.getPath());
        }
    }
}

File: src/test/java/cn/enaium/cf4m/test/config/ConfigTest.kt
package cn.enaium.cf4m.test.config

import org.junit.jupiter.api.Assertions
import cn.enaium.cf4m.CF4M.INSTANCE

/**
 * @author Enaium
 */
class ConfigTest {
    init {
        Assertions.assertNotEquals(0, INSTANCE.config.all.size)
        for (configProvider in INSTANCE.config.all) {
            println(configProvider.name + "|" + configProvider.path)
        }
    }
}